### ISSUE TITLE
Text formatting can be applied but not removed

### ISSUE THREAD

**Issue by Niretech** [davidroyer/vue2-editor#309]
### Issue
Text can be highlighted and formatting can be applied with the editor (e.g. bold, italic, bullet list etc.) but formatting can not be removed.

### My configuration
I am using vue editor in multiple places and this works without an issue - however, in a modal popup I am getting the issue where I can format text but I can not remove the formatting. If I click the button 'bold' for example, it will remain highlighted and won't remove the bold styling from the v-model value. All of my editors have a unique ID.

```
<vue-editor
    v-model="myValue"
    @input="updateValue"
    :editorToolbar="customToolbar"
    :placeholder="placeholder"
    :id="id"
    :disabled="disabled"
    class="bg-gray-50 w-full mock-input border border-transparent"
></vue-editor>
```

Input calls an event which emits myValue but that's irrelevant for this scenario as I have tested v-model with a local `test` data prop and the issue is the same - the input event will fire if I click a button to remove the formatting, however the value remains the same. If I hti CTRL+B for example, bold styling will successfully be removed, however I need these buttons to work correctly.

**Comment by henriquebarrosx** [comment]
I'm having the same problem!!! It's used on one page working fine, but when I tried to use it on another page, I didn't have the same luck.

**Comment by henriquebarrosx** [comment]
After a little break, I finally found how to solve the problem! For some reason the Vue2-editor take a few milliseconds to load by completely. Comparing the another implementation that I have in another page, I realized that there is a loading event to fetch data from specific form, and this event was that avoided the vue2-editor issue. It's a kind of workaround, but was a nice solution to fix the problem.
